js/index.js

```diff
@@ -0,0 +1,11 @@
+document.querySelector('.menu-bars').addEventListener('click', () => {
+  const menuStyle = document.querySelector('#desktop-nav').style.display;
+
+  if (menuStyle === 'none' || menuStyle === '') {
+    document.querySelector('#desktop-nav').style.display = 'block';
+    document.querySelector('#desktop-nav ul').style.display = 'block';
+  } else {
+    document.querySelector('#desktop-nav').style.display = 'none';
+    document.querySelector('#desktop-nav ul').style.display = 'none';
```

Comment:
- [ ]  As the previous code reviewer pointed out. please make sure that when the user clicks on any of the options the mobile menu disappears and redirects to the specific section on the page. kindly check the video I point it out.
https://www.loom.com/share/923543a959434d6aad239234466ee50f